README.md

```diff
@@ -40,17 +40,17 @@ Designing in the Browser is a methodology that directly uses the browser as the
 Benefits
 --------
 
-- Requires minimal skills to get started--just HTML
+- Requires minimal skills to get started (Just HTML)
 - Creates accessible prototypes
-- Once learned, much faster than any drag-and-drop tool
-- Create dynamic designs that flex and change depending on the width of the screen
+- Once learned,then it provides Experience of much faster than any drag-and-drop tool.
+- Create dynamic designs that flex & change depending on the width of the screen
 - Create designs that interact and behave exactly as deployed
 - View desktop and mobile versions of a design at the same time
-- Make design decisions based on actual browser behavior, rather than just assumptions in our designs. Think of it as deciding in the browser.
+- It Makes design decisions based on the actual browser behavior, rather than just assumptions in our designs. Think of it as deciding in the browser.
 - Allows us to test early and get feedback on reality—not assumptions from abstracted tool output. As soon as a single piece of design is in the browser, it can be tested.
 - Sets expectations early and allows the design to grow iteratively and organically.
-- Code used in prototypes can serve as a basis for production ready code.
-- Ultimately, it allows us to put content over form, embrace the fluidity of the web and the diaspora of devices, and focus on systems of design components instead of pages.
+- The Code used in prototypes can serve as a basis for a production ready code.
```

Comment:
I like the original statement better. Your suggested change is not grammatically correct.